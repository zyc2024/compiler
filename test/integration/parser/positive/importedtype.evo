

import K 

type Agg = {
    K.A.T[] stuff;
}

Agg createNewAgg (K.A.T[] s) {
    return Agg(stuff=s);
}