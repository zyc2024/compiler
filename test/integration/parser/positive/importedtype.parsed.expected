(
    ((import K))
    (
        (Agg ((stuff ([] (. (. K A) T))))    )
        (createNewAgg ((s  ([] (. (. K A) T))   )) (Agg) (
            (return (Agg ((= stuff s)) ) )
        ) )
    )
)